mobile/library/swift/YAMLValidatingEngineBuilder.swift

```diff
@@ -0,0 +1,15 @@
+/// An engine builder that validates the YAML configuration and asserts on failure.
+/// Not part of the stable public API.
+///
+/// - returns: The `EngineBuilder`.
+@_spi(YAMLValidation)
+public func YAMLValidatingEngineBuilder() -> EngineBuilder {
```

Comment:
Even though this is `public`, it's marked as `@_spi(YAMLValidation)` so that it's not available in the public API for users who `import Envoy`.